acceptance/acceptance_test.go

```diff
@@ -917,10 +936,14 @@ func testAcceptance(
 								volumeRoot = `c:\`
 								slash = `\`
 							}
-							readBuildpackTgz = h.CreateTGZ(t, filepath.Join(bpDir, "read-volume-buildpack"), "./", 0755)
-							readWriteBuildpackTgz = h.CreateTGZ(t, filepath.Join(bpDir, "read-write-volume-buildpack"), "./", 0755)
 
 							var err error
+
```

Comment:
nit: rm line?

Reply:
Happy to, the thinking was that the `err` defined here is used four times in the file. That's not really different to reassigning an `err` first assigned with `:=` though. Seems like a little for one commit, so let's figure out what other changes we want to make.

Reply:
I found a cleanup issue, so removed this with that commit